package.json

```diff
@@ -26,22 +26,53 @@
 		"xo"
 	],
 	"dependencies": {
+		"@atom/babel7-transpiler": "^1.0.0-1",
+		"@babel/preset-env": "^7.12.1",
+		"@babel/preset-typescript": "^7.12.1",
 		"atom-linter": "^10.0.0",
 		"atom-package-deps": "^5.0.0",
 		"eslint-rule-documentation": "^1.0.0",
 		"load-json-file": "^5.1.0",
 		"p-props": "^1.0.0",
-		"pkg-dir": "^3.0.0",
+		"pkg-dir": "^5.0.0",
```

Comment:
TypeScript definitions were introduced in v4, so I updated to the current version